Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.11-slim@sha256:39cecc9cde774f9209ad26a0ab2dc4f5d10ba92d2913a835cebea3b402af8e8b
+
+# Configuración Poetry (https://python-poetry.org/docs#ci-recommendations)
+ENV POETRY_VERSION=1.3.2
+
+ENV VIRTUAL_ENV=/opt/poetry-venv
+RUN python3 -m venv $VIRTUAL_ENV
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+
+RUN $VIRTUAL_ENV/bin/pip install "poetry==$POETRY_VERSION"
```

Comment:
¿Por qué instalas una versión específica de poetry?

Reply:
En el issue #30 hago referencia al uso de un entorno virtual en el docker y los motivos

Reply:
A estas alturas deberíais saber que lo único que tengo en consideración es la documentación que aportéis.
En todo caso, no creo que sea necesario.

Reply:
Se refiere a que no sería necesario hacer uso de un entorno virtual??
Para el tema de la documentación, deberia de hacer un fichero .md donde explique lo que estoy haciendo comando a comando?

Reply:
No, ¿para qué?